arangod/Agency/FailedFollower.cpp

```diff
@@ -369,7 +370,7 @@ bool FailedFollower::start(bool& aborts) {
 
   slice = result.get(std::vector<std::string>(
       {agencyPrefix, "Supervision", "Health", _to, "Status"}));
-  if (!slice.isString() ||
+  if (slice.isString() &&
```

Comment:
This changes the logic: If the slice is not a string (for example, does not exist), we used to see the log message, but now we would no longer see it. Is this intentional?